instrumentation/oppia_instrumentation_test.bzl

```
@@ -0,0 +1,40 @@
+"""
+Instrumentation macros to setup up end-to-end tests.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+def oppia_instrumentation_test(
+        name,
+        srcs,
+        deps):
+    """
+    Creates library, binary and a instrumentation test for each test suite.
+
+    Args:
+        name: str. The class name of the Test suite or the test file.
+        srcs: list of str. List of test files under instrumentation module.
+        deps: list of str. The list of dependencies needed to build and run the library.
+    """
+    kt_android_library(
+        name = name + "_lib",
```

```suggestion
        name = "%s_lib" % name,
```

Prefer string interpolation. Ditto elsewhere in this file.

---

Done